manifests/baremetal/keepalived.conf.tmpl

```diff
@@ -33,3 +33,20 @@ vrrp_instance {{`{{.Cluster.Name}}`}}_DNS {
         {{`{{ .Cluster.DNSVIP }}`}}/{{`{{ .Cluster.VIPNetmask }}`}}
     }
 }
+
+{{`{{if .Cluster.APIProvisioningVIP}}`}}
+vrrp_instance {{`{{.Cluster.Name}}`}}_PROVISION {
+    state BACKUP
+    interface {{`{{ .VRRPProvInterface }}`}}
+    virtual_router_id {{`{{ .Cluster.APIProvisioningVirtualRouterID }}`}}
+    priority 40
```

Comment:
If we prefer that the bootstrap node (as long as it exists) will hold this VIP, shouldn't we set a higher priority?

Reply:
Yes. Thanks for catching that.